lib/components/molecules/DateSelector/DateSelector.js

```diff
@@ -0,0 +1,203 @@
+// @flow
+/**
+ *
+ * DateSelector
+ *
+ */
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import classNames from 'classnames';
+import styles from './DateSelector.style';
+import type { Props } from './types';
+import Select from '../../atoms/Select';
+import Label from '../../atoms/Label';
+
+const monthOptions = [
+  'January',
+  'Febuary',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+const DateSelector = (props: Props): Node => {
+  const { className, format, startDate, endDate } = props;
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [newStartDate, setNewStartDate] = useState(
+    new Date(new Date().setFullYear(new Date().getFullYear() - 100))
+  );
+  const [newEndDate, setNewEndDate] = useState(
+    new Date(new Date().setFullYear(new Date().getFullYear() + 50))
```

Comment:
Where did you find this 50 or 100 number?

Reply:
Those were for the default start and end date